test/service_instance_test/service_instance_test.go

```diff
@@ -125,7 +128,10 @@ var _ = test.DescribeTestsFor(test.TestCase{
 
 func blueprint(ctx *common.TestContext, auth *common.SMExpect) common.Object {
 	serviceInstance := prepareServiceInstance(ctx, auth, fmt.Sprintf(`{"%s":"%s"}`, TenantIdentifier, TenantValue))
-	ctx.SMRepository.Create(context.Background(), serviceInstance)
+	_, err := ctx.SMRepository.Create(context.Background(), serviceInstance)
+	if err != nil {
```

Comment:
Expect err to not have occurred instead ?

Reply:
At this point the fail handler is not yet registered. Not quite sure why.